[gamemodes]/[play]/play/play_vehicles.lua

```diff
@@ -0,0 +1,98 @@
+local vehicleTimers = {}
+local playerVehicles = {}
+local vehiclesToSpawn = {}
+
+function createVehicles()
+	for vehicleID = 1, #vehicleSpawns do
+		createPlayVehicle(vehicleSpawns[vehicleID])
+	end
+end
+
+function createPlayVehicle(vehicleData)
+	local modelID, posX, posY, posZ, rotX = vehicleData[1], vehicleData[2], vehicleData[3], vehicleData[4], vehicleData[5]
```

Comment:
You can use [unpack](https://www.lua.org/manual/5.1/manual.html#pdf-unpack) to easily get the table elements.
```suggestion
	local modelID, posX, posY, posZ, rotX = unpack(vehicleData)
```